src/views/ConfusionMatrix/ConfusionMatrixView.tsx

```diff
@@ -0,0 +1,118 @@
+import { INITIALIZE_UNITS, useSelectedUnitIds } from 'contexts/UnitSelection/UnitSelectionContext'
+import { FunctionComponent, useCallback, useEffect, useMemo, useState } from 'react'
+import BottomToolbar from 'views/UnitSimilarityMatrix/BottomToolbar'
+import { sortIds } from 'views/UnitsTable/UnitsTableView'
+import MatrixWidget from '../UnitSimilarityMatrix/MatrixWidget'
+import { ConfusionMatrixViewData } from './ConfusionMatrixViewData'
+
+type Props = {
+    data: ConfusionMatrixViewData
+    width: number
+    height: number
+}
+
+const defaultRange: [number, number] = [0, 1]
+
+export type HoveredInfo = {
+    unitId1: number | string
+    unitId2: number | string
+    value: number | undefined
+}
+
+const ConfusionMatrixView: FunctionComponent<Props> = ({ data, width, height }) => {
```

Comment:
This is the new ConfusionMatrixViwe component. It is modeled after UnitSimilarityMatrixView. It uses the same MatrixWidget